### ISSUE TITLE
Sample data

### ISSUE THREAD

**Issue by Brynjulf** [equinor/esv-intersection#35]
We need sample data that we can use for testing purposes as well as displaying stories in storybook

We can use Volve data seeing as it is made available for everyone. We do need to figure out if we can expose the data from our repository or if we need to keep it locally. I remember they had something in the license about the need for keeping track of who downloaded the data.

related to #140

**Comment by andreabrambilla** [comment]
As mentioned in the slack channel, the license is a normal Creative Common, we should be able to use the data with no string attached

**Comment by SvSven** [comment]
See: https://www.equinor.com/en/how-and-why/digitalisation-in-our-dna/volve-field-data-village-download.html

It is licensed as CC BY-NC-SA 4.0 but they do indeed include something about keeping track of who downloaded it.

From the FAQ page:

> **Personal data**
The «Volve data village» will be accessible in for approximately two years from date of release 14.06.18. In order to comply with the agreements relevant for the release of the data, it is necessary for Equinor to maintain control of who has downloaded the data. Consequently, name, email, company and date of download will be registered when asking for access to the data, and retained for the necessary time-period. For further information, please see Equinor’s privacy policy available on equinor.com.

**Comment by andreabrambilla** [comment]
@thorvaldj Do you have any comment about this?

**Comment by andreabrambilla** [comment]
Had a chat with Thorvald, in theory there should be no problem, but I have to double check with a few other people first

**Comment by andreabrambilla** [comment]
So, the license for Volve will be updated in a couple of weeks. One important point is that the data can be used in commercial contexts, but it cannot be sold. That is something the the MIT license would allow, so we cannot just take a piece of the dataset and put it on our repo.
One possibility could be to make it clear that the data has a different license.
Another thing to look into is how the "adapted" data is treated. In other words, if we take a projection of a volume on a 2d slice, that may count as "adapted" data, and that may be compatible with MIT.

Long story short, we need to read the new license carefully. Until the new license is out, let us not put any piece of data from Volve on this repo

**Comment by miwegmann** [comment]
What do you mean with "projection"? So far I agree not to expose the Volve data yet

**Comment by andreabrambilla** [comment]
The new license for Volve is out. We have to find a way to make it available in the Storybook (or in an interactive demo if we decide to host one).
The Volve data can be probably programmatically fetched from Azure, but we would have to apply some transformation to it, it may require a lot of efforts.
A possibility can be to pre-compute the transformed data and host it somewhere publicly accessible, with the same license as Volve

**Comment by andreabrambilla** [comment]
superseded by #154
